ClientApp/src/components/Layout.js

```diff
@@ -1,17 +1,30 @@
 import React, { Component } from 'react';
 import { Container } from 'reactstrap';
 import { NavMenu } from './NavMenu';
+import { Chat } from './Chat';
 
 export class Layout extends Component {
```

Comment:
Maybe you should change the Layout class to a functional component to be consistent in our project.